AutomatedLabWorker/AutomatedLabWorkerInternals.psm1

```diff
@@ -132,3 +132,80 @@ Function Test-IpInSameSameNetwork
 }
 
 #endregion Test-IpInSameSameNetwork
+
+#author Iain Brighton, from here: https://gist.github.com/iainbrighton/9d3dd03630225ee44126769c5d9c50a9
+function Get-RequiredModulesFromMOF {
```

Comment:
Isn't that function better in `AutomationLab.Common`. We have a `DscHelper` folder anlread there.

Reply:
Good idea. I'll do that